__tests__/app.test.js

```diff
@@ -73,6 +73,26 @@ describe("/api/reviews/review_id", () => {
           created_at: expect.any(String),
           votes: 1,
           review_id: 1,
+          comment_count: "0"
```

Comment:
Doing a new separate test for the comment count is exactly the right thing to do 👍 but we shouldn't update our old test. The old test should still pass even with the new addition to the review object (which is tested for by the new test).